examples/opensora_hpcai/opensora/pipelines/train_pipeline.py

```diff
@@ -17,6 +19,21 @@
 logger = logging.getLogger(__name__)
 
 
+@jit_class
```

Comment:
is there any exposed no_grad API in MS? why use internal `_no_grad`

Reply:
There is no public API for `_no_grad` yet.